cypress_test/integration_tests/desktop/impress/editable_area_spec.js

```diff
@@ -22,7 +22,7 @@ function selectTextShape(i) {
             cy.cGet('body').click(XPos, YPos);
         });
 
-    cy.cGet('#map .html-object-section').should('have.length', 9);
+    cy.cGet('[id^="test-div-shape-handle-"]').should('have.length', 6); // 3 of the handles are not visible because of the side bar.
```

Comment:
hmm, wouldn't be better to hide sidebar to be sure we are having all the needed handles?